google_drive/tests/unit/test_docs.py

```diff
@@ -0,0 +1,240 @@
+""" Tests for google drive components """
```

Comment:
I'm a little confused as to the different between google drive and google docs -- it seems we have a google drive component that supports google calendar and google docs and we split up most of the functionality between these two concepts.  I see calendar tests in this file -- maybe we should create a google_drive/tests/unit/test_calendar.py module to follow suit with the rest of the code?